lib/wovnrb.rb

```diff
@@ -163,10 +116,10 @@ def put_back_noscripts!(output, noscripts)
     # @param {Nokogiri::HTML5::Document} body The HTML body to check.
     #
     # @returns {Boolean} True is the HTML body is an AMP, false otherwise.
-    def is_amp_page?(body)
-      html_attributes = body.xpath('//html')[0].try(:attributes) || {}
+    def amp?(html_body)
```

Comment:
I prefer 'is_amp_page?' or 'amp_page?' over 'amp?'

Reply:
ok, so I will change into `amp_page?` (`is_*` is not common in ruby)

Reply:
thanks